### ISSUE TITLE
Slot definition has to have `mappings` but the docs don't mention it

### ISSUE THREAD

**Issue by samsucik** [RasaHQ/rasa#10216]
Looking at the docs for slots, the first example [here](https://rasa.com/docs/rasa/next/domain#slots) shows a slot with no `mappings` defined. While all examples below under Slot Types do include `mappings`, this one (+ the 2 examples under Slots and Conversation Behavior) is at the top, hence the most likely to get copy-pasted into one's domain file. Then, one hits an error when running `rasa train`:
```bash
YamlValidationException: Failed to validate '/Users/sams/qa/dynamic-form-behaviour/domain.yml'. Please make sure the file is correct and all mandatory parameters are specified. Here are the errors found during validation:
  in /Users/sams/qa/dynamic-form-behaviour/domain.yml:51:
      Cannot find required key 'mappings'. Path: '/slots/slot_2'
```

**Comment by melindaloubser1** [comment]
@ancalita FYI for the docs PR - I just picked up on this too 😅

**Comment by ancalita** [comment]
@samsucik The changes you flagged are addressed by the current PR under review in the `domain.mdx` file : https://github.com/RasaHQ/rasa/pull/10011/files

Please let me know if the PR covers this issue or if there are other places that need updating?

**Comment by ancalita** [comment]
Closing this issue, since it was addressed in #10011 which just got merged.
